core/src/main/java/org/infinispan/util/concurrent/BlockingManagerImpl.java

```diff
@@ -370,6 +370,19 @@ public <V> ScheduledBlockingCompletableStage<V> scheduleRunBlocking(Supplier<V>
       return scheduledStage;
    }
 
+   @Override
+   public ScheduledBlockingCompletableStage<Void> scheduleRunBlockingAtFixedRate(Runnable runnable, long initialDelay, long period, TimeUnit unit, Object traceId) {
```

Comment:
One thing to note is do we really want AtFixedRate? Is FixedDelay okay, that one is usually better as it guarantees you can't have concurrent runnables running at the same time. The drawback is the scheduling is not always done at the same offset.

Reply:
Good point. I used AtFixedRate in place of FixedDelay since the way the EAP integration works. AtFixedRate sounds the way RedHat Insights works, since we can imagine a user opens the Insights web page and expects an update at a given hour of time. Anyway I'll ask if we can switch to FixedDelay here to the Insight experts.
Unfortunately so far I haven't had the opportunity to have any contact with any Insight experts from the company, thus all I’ve done so far I just deducted looking at the Insights Java client and at EAP Insight integration :(